src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -188,10 +209,18 @@ public void copyFile(File file) {
             ZipUtil.pack(file, tmp);
             byte[] archive = IOUtils.toByteArray(new FileInputStream(tmp));
 
-            addShellStep(String.format(
-                    "base64 --decode << ENDOFFILE > archive.zip && unzip -o archive.zip \n%s\nENDOFFILE",
-                    new String(Base64.encodeBase64Chunked(archive))
-            ));
+            if (SystemUtils.IS_OS_WINDOWS) {
+                if (!(controller instanceof LocalController)) {
+                    // TODO: Make it work for RemoteJenkinsController like in Unix (below)
+                    throw new AssumptionViolatedException("Copying files in Windows is only supported if a LocalController is in use. Test will be skipped.");
```

Comment:
Since (this part of) this PR is strictly adding functionality, I think we can approve it as is, and use separate PRs to expand the coverage to more combinations such as Windows + remote controller, or to different OSs locally vs. remotely.